packages/connection/legacy/class-jetpack-xmlrpc-server.php

```diff
@@ -747,7 +750,10 @@ public function unlink_user() {
 		 * @param string $data Optional data about the event.
 		 */
 		do_action( 'jetpack_event_log', 'unlink' );
-		return Connection_Manager::disconnect_user();
+		return Connection_Manager::disconnect_user(
+			empty( $user_id ) ? null : $user_id,
```

Comment:
Yes, you're right. 

I was doing that before I realized I was getting an empty array instead of  `null`, but for some reason, I steped back. But look at it now it seems we could pass the empty array over as well.